### PR TITLE
Update French Banner Text Size

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- French banner text is too long to be 6xl in md viewports see images for changes
- Button text aligned center
Before:
image

After:
image

### Additional Notes

### CODE REVIEW

File: frontend/src/pages/home.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const Home: FC = () => {
       <Layout contained={false}>
         <Container className="mb-8 md:mb-12">
           <HeroBanner imageProps={{ className: 'md:object-right-bottom', src: landingPageImage }}>
-            <h1 className="mb-2 font-display text-4xl font-bold text-primary-700 md:mb-4 md:text-6xl">
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:87]: It should be `md:text-4xl lg:text-6xl` for both languages.

File: frontend/src/pages/learn/index.tsx [hunk 2]
```diff
@@ -188,11 +190,11 @@ const Learn: FC = () => {
         ]}
       >
         <HeroBanner imageProps={{ className: 'md:object-right-bottom', src: learnBannerImage }}>
-          <h1 className="mb-2 font-display text-4xl font-bold text-primary-700 md:mb-4 md:text-6xl">
+        <h1 className={`mb-2 font-display text-4xl font-bold text-primary-700 md:mb-4 ${locale === 'en' ? 'md:text-6xl' : 'md:text-4xl lg:text-6xl'}`}>
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:193]: It should be `md:text-4xl lg:text-6xl` for both languages.

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/src/pages/learn/index.tsx (hunk 2 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/learn/index.tsx]: Fixed

File: frontend/src/pages/learn/index.tsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ import rulesOfThumbForPublicPensionsSmImage from '../../../public/assets/rules-o
 import { HeroBanner } from '../../components/HeroBanner'
 import Layout from '../../components/Layout'
 import { getDCTermsTitle } from '../../utils/seo-utils'
+import { useRouter } from 'next/router'
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:23]: `useRouter` is not used, should be removed

Comment (CurtisUnderwood) [reply frontend/src/pages/learn/index.tsx]: Fixed

File: frontend/src/pages/learn/index.tsx [hunk 4]
```diff
@@ -83,6 +83,7 @@ const LearnSection: FC<LearnSectionProps> = ({ cards, desciption, id, title }) =
 
 const Learn: FC = () => {
   const { t } = useTranslation('learn')
+  const { locale } = useRouter()
```
[the remark this answers is not in the document]
Comment (CurtisUnderwood) [reply frontend/src/pages/learn/index.tsx]: 👍

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**